index.scss

```diff
@@ -94,14 +94,6 @@ button:focus {
   .main-content .main-page-row .column-container .ways {
     max-width: 328px;
   }
-  .main-content .main-page-row .column-container .slider,
```

Comment:
Can we maybe fix this file add a new card to refactor this file to use the Sass powers and nesting? It looks very very bad😳

Reply:
I 100% agree Akmal! Good thing, I already started working on refactoring scss in seperate scss modules. I did a big part of this already in a different PR here: https://github.com/deriv-pocs/deriv-developers-portal/pull/59